features/zones/Create.feature

```diff
@@ -97,14 +112,34 @@ Feature: zones / create
       """
       body:
         zone:
-          enabled: true
+          enabled: !!js/undefined
         zoneInsight:
           subscriptions:
             - connectTime: '2020-07-28T16:18:09.743141Z'
               disconnectTime: !!js/undefined
       """
     Then the "$connected" element exists
 
+  Scenario: The zone is deleted whilst waiting to be connected and shows an error
+    Given the environment
+      """
+      KUMA_SUBSCRIPTION_COUNT: 0
+      """
+    And the URL "/provision-zone" responds with
+      """
+      body:
+        token: spat_595QOxTSreRmrtdh8ValuoeUAzXMfBmRwYU3V35NQvwgLAWIU
+      """
+    When I visit the "/zones/-create" URL
+    And I "type" "test" into the "$name-input" element
+    And I click the "$create-zone-button" element
+    And the URL "/zones/test/_overview" responds with
+      """
+      headers:
+        Status-Code: 404
+      """
+    Then the "$error" element exists
+
```

Comment:
New test to test that we show an error state on 404. If someone ever makes a similar same mistake again this would fail and cause whoever was doing it to think twice.